src/Panel/SqlLogPanel.php

```diff
@@ -52,10 +51,7 @@ public function initialize(): void
 
         foreach ($configs as $name) {
             $connection = ConnectionManager::get($name);
-            if (
-                $connection->configName() === 'debug_kit'
-                || !$connection instanceof ConnectionInterface
-            ) {
+            if ($connection->configName() === 'debug_kit') {
```

Comment:
Hmm.. 4.x allowed the `ConnectionManager` to hold connection instances which did not implement `ConnectionInterface`, for e.g. https://github.com/UseMuffin/Webservice/blob/6376dc722529d888c4750793f0b286a8acd9fd71/src/Datasource/Connection.php#L19.

Guess we killed that in 5.0 by adding `ConnnectionInterface` as return type for `ConnectionManager::get()`.